index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="div-container">
+      <section>
+        <header class="header">
+          <div class="My-Logo">
+            <h1>My Logo</h1>
+          </div>
+          <div class="header-Line-parent">
+            <div class="header-Line-1"></div>
+            <div class="header-Line-2"></div>
+            <div class="header-Line-3"></div>
+          </div>
+        </header>
+      </section>
+
+      <section class="section-1">
+        <div class="heading-1">
+          <h1 class="First-heading">
+            I'm Mariane <br />
+            Glad to see <br />
+            you!
+          </h1>
+          <span class="First-heading-description">
+            I'm a software developer! I can help you build a prduct,feature or
+            website Look through some of my work and experience! if you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </span>
+          <h3 class="Connect-heading">LET'S CONNECT</h3>
+        </div>
+      </section>
+
+      <div class="icons-div">
+        <img class="twitter icons" src="Vector (5).svg" alt="" />
+        <img class="linkedin icons" src="Linkedin.svg" alt="" />
+        <img class="medium icons" src="Vector (2).svg" alt="" />
+        <img class="github icons" src="Vector.svg" alt="" />
+        <img class="angellist icons" src="Vector (1).svg" alt="" />
+      </div>
```

Comment:
- I'd like to kindly suggest that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

Something like this 👇🏽 
```
 <ul class="icons-div">
        <li><a href = "yourtwitterhandle"><img class="twitter icons" src="Vector (5).svg" alt="" /></a></li>
        <li><a href = "yourlinkedinhandle"><img class="linkedin icons" src="Linkedin.svg" alt="" /></a></li>
        <li><a href = "yourmediumhandle"><img class="medium icons" src="Vector (2).svg" alt="" /></a></li>
        <li><a href = "yougithubhandle"><img class="github icons" src="Vector.svg" alt="" /></a></li>
        <li><a href = "yourangelisthandle"><img class="angellist icons" src="Vector (1).svg" alt="" /></a></li>
    </ul>
```
- Also, kindly include your entire icons inside the `<section class="section-1">` as it is a part of the section. 🌸